examples/truffle/package.json

```diff
@@ -4,10 +4,12 @@
   "private": true,
   "description": "Test contracts for ethcontract-rs runtime and proc macro.",
   "scripts": {
-    "build": "rimraf build && truffle build && node scripts/artifacts.js normalize",
+    "build": "rimraf build && truffle build",
     "deploy": "truffle migrate --reset --network rinkeby && node scripts/artifacts.js extract-networks",
+    "normalize": "node scripts/artifacts.js normalize",
```

Comment:
What is normalize?

Reply:
Unfortunately multiple compilations of the same truffle contracts yield different truffle artifacts and even different bytecode :warning: ! So we have to normalize the artifacts so that when compiled locally, or on the CI the result is the same.

Reply:
Cool beans!